tests/test_config_flow.py

```
@@ -238,10 +238,29 @@ def test_detect_serta_motion_perfect_name(self):
         assert bed_type == BED_TYPE_OCTO
 
+    def test_detect_octo_rc2_receiver(self, mock_bluetooth_service_info_octo_rc2):
+        """Test detection of Octo RC2 receiver - defaults to Octo for shared UUID.
+
+        Issue #73: Devices like RC2 that share the Solace UUID but don't have
+        'solace' in the name should default to Octo since it's more common.
+        """
+        bed_type = detect_bed_type(mock_bluetooth_service_info_octo_rc2)
+        assert bed_type == BED_TYPE_OCTO
+
+    def test_detect_solace_bed(self, mock_bluetooth_service_info_solace):
```

Ruff (0.14.11)

Redefinition of unused `test_detect_solace_bed` from line 89: `test_detect_solace_bed` redefined here

(F811)